history.txt

```diff
@@ -1,6 +1,34 @@
 OMERO version history
 =====================
 
+5.1.0 (March 2015)
+------------------
+
+A full, production-ready release of OMERO 5.1.0; updating the Data Model to
+the January 2015 schema, including support for units and new more
+flexible user-added metadata; and introducing new user features, new supported
+formats and many fixes and performance improvements:
+
+-  support for units throughout the Data Model allowing for example, pixel
+   sizes for electron microscopy to be stored in nanometers rather than being
+   set as micrometers
+-  new, searchable key-value pairs annotations for adding experimental
+   metadata
+-  improved workflow for rendering settings in the UI and parity between the
+   clients
+-  import images to OMERO from ImageJ and save ROIs and overlays from ImageJ
+   to OMERO
+-  'Import As' for group owners as well as administrators
+-  improved performance for moving and deleting data
+-  import times are much improved for large datasets such as HCS and SPIM data
+-  improved performance for many file formats and new supported formats via
+   Bio-Formats (now over 140)
+-  new OMERO.mail feature lets admins configure the server to email users
+-  support for limiting the download of original files
```

Comment:
Why would you want to do that? Maybe add "...for public facing OMERO.web deployments"?